budgetportal/utils.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from requests.exceptions import HTTPError, ConnectionError
+
+
+def get_budget_resources(year, dept_name):
```

Comment:
I think instead of being a function in a new module, this should be a method of the Department class in `models`.